app/src/main/java/org/schabi/newpipe/settings/custom/NotificationActionsPreference.java

```diff
@@ -85,19 +85,22 @@ private void setupActions(@NonNull final View view) {
     ////////////////////////////////////////////////////////////////////////////
 
     private void saveChanges() {
-        final SharedPreferences.Editor editor = getSharedPreferences().edit();
+        if (compactSlots != null && notificationSlots != null) {
```

Comment:
Can you add the `@Nullable` to the member fields like you did for `menuItemToFilterName`